src/codeflare_sdk.egg-info/PKG-INFO

```diff
@@ -1,4 +1,4 @@
 Metadata-Version: 2.1
-Name: codeflare-sdk
+Name: codeflare_sdk
```

Comment:
Is this change required?

Reply:
Why is the egg-info not in the .gitignore in the first place?

Reply:
@Ygnas Very good question!
I smell a handy PR :)